### PR TITLE
Add INSTALL doc file

### PR DESCRIPTION
**Pull request by CharliePoole**
Related to #325 

@mikkelbu @immeraufdemhund Would you take a look at the new file and suggest any changes that seem to be needed?

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# TestCentric Installation
+
+**TestCentric** is released in two package formats, **zip** and **chocolatey**, both available from the [GitHub project](https://github.com/TestCentric/testcentric-gui/releases) site. The **chocolatey** package is also available at [chocolatey.org](https://chocolatey.org/testcentric-gui).
```
Comment (mikkelbu) [inline INSTALL.md]: The link `https://chocolatey.org/testcentric-gui` should be `https://chocolatey.org/packages/testcentric-gui`

File: INSTALL.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+## Chocolatey Installation
+
+This is the recommended way to install chocolatey on a developer machine. It provides the best user experience and creates shim commands for both the standard and the experimental GUIs so they may be invoked from any directory.
```
Comment (mikkelbu) [inline INSTALL.md]: "install chocolatey" => "install TestCentric".

File: README.md [hunk 3]
```diff
@@ -18,10 +18,12 @@ The initial release is based on the layout and feature set of the of the origina
 
 **TestCentric** is released in two package formats, **zip** and **chocolatey**, both available from the [GitHub project](https://github.com/TestCentric/testcentric-gui/releases) site. The **chocolatey** package is also available at [chocolatey.org](https://chocolatey.org/testcentric-gui).
 
-The **chocolatey** package provides the best user experience and is the recommended way to install **TestCentric**. Follow the directions at [chocolatey.org](https://chocolatey.org/testcentric-gui). If you wish to install any NUnit engine extensions for use with the GUI, simply install them using `choco.exe` in the same way as the GUI.
+The **chocolatey** package provides the best user experience and is the recommended way to install **TestCentric**. Follow the directions at [chocolatey.org](https://chocolatey.org/testcentric-gui) or see the [INTALL](./INSTALL.md) document. If you wish to install any NUnit engine extensions for use with the GUI, simply install them using `choco.exe` in the same way as the GUI.
```
Comment (mikkelbu) [inline README.md]: Similar to the comment about https://chocolatey.org/testcentric-gui in INSTALL.md

File: INSTALL.md (hunk 1 above)
Comment (CharliePoole) [reply INSTALL.md]: Good catch. Especially since I copied the link from someplace, which I should now also change!

**Pull request merged**